src/libs/workspace-utils.ts

```diff
@@ -15,19 +15,21 @@ export const isKnownCloudProvider = (x: unknown): x is CloudProvider => {
   return (x as string) in cloudProviderTypes;
 };
 
-export interface BaseWorkspaceInfo {
+interface BaseWorkspaceInfo {
```

Comment:
This should only be used as a convenience for declaring fields common to both `AzureWorkspaceInfo` and `GoogleWorkspaceInfo`. Anything else should use the union type `WorkspaceInfo`.